src/main/java/subway/stations/SubwayStations.java

```diff
@@ -30,6 +30,73 @@ public int getObjectId() {
         }
     }
 
+    public List<Station> dijkstrasAlgorithm(Station source, Station destination){
+
+        Set<Station> visitedStations = new HashSet<>();
+        Set<Station> unvisitedStations = new HashSet<>();
+
+        Station currentSource = source;
+        int shortestDistance = Integer.MAX_VALUE;
```

Comment:
What is `shortestDistance` for?

Reply:
I use it in creating the path. I check which of the destination's connections has a distance from the origin that is equal to the shortest distance. That is the station that I work backwards from.

Reply:
You've got `shortestDistance` and `shortestDistanceInSet`